docs/tests-github.md

```diff
@@ -0,0 +1,55 @@
+---
+category: DevelopInDepth
+
+---
+# Using GitHub Actions
+
+[Github Action](https://github.com/features/actions) makes it easy to automate all your software workflows, now with world-class CI/CD. Build, test, and deploy your code right from GitHub. Make code reviews, branch management, and issue triaging work the way you want.
+
+
+## GitHub Action Scripts
+
+All the GitHub Action build file are locate in `.github/workflows/*.yml`
+
+Each script is triggered by its own conditions, for more details see [Github Action Docs](https://docs.github.com/en/actions)
+
+Matomo uses GitHub Action to automatically run its build by various triggers.
+- [Build Tracker JS] Trigger by the comment `build js` into the pull request. That will compress js/piwik.js into matomo.js and piwik.js
+- [Build VUE] Auto trigger, it will build VUE file and commit back to the PR
+- [Composer Update] Auto trigger by cron，it will run `composer update` and commit back to the repo
+- [PHPCS check] Auto trigger, checking PHPCS code quality if not valid will return error details.
+- [Inactive PR] after 14 days, inactive PR will mark as state
+
+(New scripts should be documented in the list above.)
+
+## Create a new GitHub Action script
+
+To create a new GitHub Action script please read [Quickstart for GitHub Actions](https://docs.github.com/en/actions/quickstart)
+
+We recommend forking the Matomo project and running actions on your own pipeline first. Also, please check the following list before getting started:
+
+### Security
+
+There is a chance the action scripts getting hacked, paying attention to security is of the upmost importance when you create a new GitHub Action script. 
+
+It could lead a GitHub token lost and write access to our code. [More details](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#stealing-the-jobs-github_token)
```

Comment:
```suggestion
It could lead to a GitHub token loss and write access to our code. [More details](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#stealing-the-jobs-github_token)
```